source/SpaceportPanel.cpp

```diff
@@ -24,14 +24,16 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include "Planet.h"
 #include "PlayerInfo.h"
 #include "Random.h"
+#include "Screen.h"
 #include "UI.h"
 
 using namespace std;
 
 
 
 SpaceportPanel::SpaceportPanel(PlayerInfo &player)
-	: player(player), port(player.GetPlanet()->GetPort()), ui(*GameData::Interfaces().Get("spaceport"))
+	: player(player), port(player.GetPlanet()->GetPort()), ui(*GameData::Interfaces().Get(Screen::Width() < 1280 ?
+		"spaceport (small screen)" : "spaceport"))
```

Comment:
Not indented further in PlanetPanel.cpp, and let's keep the item on one line if possible.
```suggestion
	: player(player), port(player.GetPlanet()->GetPort()),
	ui(*GameData::Interfaces().Get(Screen::Width() < 1280 ? "spaceport (small screen)" : "spaceport"))
```